services/credentials-service/prisma/migrations/20230304094253_migrate_mongodb_to_psql/migration.sql

```diff
@@ -0,0 +1,68 @@
+-- CreateEnum
+CREATE TYPE "VCStatus" AS ENUM ('PENDING', 'ISSUED', 'REVOKED');
+
+-- CreateTable
+CREATE TABLE "VC" (
+    "id" TEXT NOT NULL,
+    "type" TEXT NOT NULL,
+    "subject" TEXT NOT NULL,
+    "issuer" TEXT NOT NULL,
+    "status" "VCStatus" NOT NULL DEFAULT 'PENDING',
+    "credential_schema" TEXT NOT NULL,
+    "unsigned" JSONB NOT NULL,
+    "signed" JSONB,
+    "created_at" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updated_at" TIMESTAMP(3) DEFAULT CURRENT_TIMESTAMP,
+
+    CONSTRAINT "VC_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "VCV2" (
```

Comment:
Since the APIs are not connected with the registry right now, so we have a postgres instance associated wih the credential service to save the issued credentials in the postgres instance. This table stores those credentials.

Reply:
we need to fix the table name

Reply:
are these fixed? @techsavvyash